### PR TITLE
Refine some db codes

### PR DESCRIPTION
**Pull request by sneaxiy**
Remove unnecessary codes and imports.

### CODE REVIEW

File: python/runtime/db_writer/maxcompute.py [hunk 1]
```diff
@@ -11,14 +11,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from odps import ODPS, tunnel
 from runtime.db_writer.base import BufferedDBWriter
 
 
 class MaxComputeDBWriter(BufferedDBWriter):
     def __init__(self, conn, table_name, table_schema, buff_size):
```
Comment (shendiaomo) [inline python/runtime/db_writer/maxcompute.py:28]: Add a docstring for this function?

File: python/runtime/db_writer/mysql.py [hunk 2]
```diff
@@ -16,15 +16,15 @@
 
 class MySQLDBWriter(BufferedDBWriter):
     def __init__(self, conn, table_name, table_schema, buff_size):
```
Comment (shendiaomo) [inline python/runtime/db_writer/mysql.py:28]: ditto

File: python/runtime/maxcompute.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 # Here we use the sdk to operate the database
 class MaxCompute:
     @staticmethod
-    def connect(database, user, password, host, auth=""):
+    def connect(database, user, password, host):
```
Comment (shendiaomo) [inline python/runtime/maxcompute.py:22]: ditto

File: python/runtime/db_writer/mysql.py [hunk 4]
```diff
@@ -16,15 +16,15 @@
...
-        return super().__init__(conn, table_name, table_schema, buff_size)
-
-    def flush(self):
-        statement = '''insert into {} ({}) values({})'''.format(
+        super().__init__(conn, table_name, table_schema, buff_size)
+        self.statement = '''insert into {} ({}) values({})'''.format(
             self.table_name, ", ".join(self.table_schema),
             ", ".join(["%s"] * len(self.table_schema)))
+
+    def flush(self):
```
Comment (shendiaomo) [inline python/runtime/db_writer/mysql.py:34]: ditto

File: python/runtime/db_writer/maxcompute.py (hunk 1 above)
Comment (sneaxiy) [reply python/runtime/db_writer/maxcompute.py:28]: Done.

File: python/runtime/maxcompute.py (hunk 3 above)
Comment (sneaxiy) [reply python/runtime/maxcompute.py:22]: Done.

File: python/runtime/db_writer/mysql.py (hunk 4 above)
Comment (sneaxiy) [reply python/runtime/db_writer/mysql.py:34]: Done.

File: python/runtime/db_writer/mysql.py (hunk 2 above)
Comment (sneaxiy) [reply python/runtime/db_writer/mysql.py:28]: Done.

**Review by shendiaomo — APPROVED** [review]
LGTM👍👍

**Pull request merged**